jupyter_collaboration/handlers.py

```diff
@@ -245,8 +245,12 @@ def on_message(self, message):
 
         if message_type == MessageType.CHAT:
             msg = message[2:].decode("utf-8")
+
             user = self.get_current_user()
-            data = json.dumps({"username": user.username, "msg": msg}).encode("utf8")
+            data = json.dumps(
+                {"sender": user.username, "timestamp": time.time(), "content": json.loads(msg)}
```

Comment:
Should `timestamp` be a UTC `datetime`?
Also, there is no need for `msg` to be deserialized and serialized again in `"content"`.